hstio/hstio.c

```diff
@@ -1037,28 +1027,26 @@ void setStorageOrder(SingleGroup * group, enum StorageOrder storageOrder)
 void copyOffsetFloatData(float * output, const float * input,
         unsigned nRows, unsigned nColumns,
         unsigned outputOffset, unsigned inputOffset,
-        unsigned outputSkipLength, unsigned inputSkipLength)
-{
+        unsigned outputSkipLength, unsigned inputSkipLength) {
     //WARNING - assumes row major storage
-    {unsigned ithRow;
 #ifdef _OPENMP
-    #pragma omp parallel for shared(output, input) private(ithRow) schedule(static)
```

Comment:
Previously `ithRow` was forced private. Now it's just private: `for (size_t ithRow = 0; ...)`